vision/src/autogluon/vision/predictor/predictor.py

```diff
@@ -41,11 +41,13 @@ class ImagePredictor(object):
         Higher levels correspond to more detailed print statements (you can set verbosity = 0 to suppress warnings).
         If using logging, you can alternatively control amount of information printed via logger.setLevel(L),
         where L ranges from 0 to 50 (Note: higher values of L correspond to fewer print statements, opposite of verbosity levels)
+    label : str, default = 'label'
+        Name of the column that contains the target variable to predict.
     """
     # Dataset is a subclass of `pd.DataFrame`, with `image` and `label` columns.
     Dataset = _ImageClassification.Dataset
 
-    def __init__(self, problem_type=None, eval_metric=None, path=None, verbosity=2):
+    def __init__(self, problem_type=None, eval_metric=None, path=None, verbosity=2, label='label'):
```

Comment:
Can we make label the first argument? That way it aligns with Tabular and Text so users can do `ImagePredictor('my_label')`